packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -166,11 +169,11 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
       {
         className: this._classNames.root,
         ref: this._buttonElement,
-        'disabled': isPrimaryButtonDisabled,
+        'disabled': isPrimaryButtonDisabled && !allowDisabledFocus,
         'aria-label': ariaLabel,
         'aria-labelledby': ariaLabelledBy,
         'aria-describedby': ariaDescribedBy,
-        'data-is-focusable': ((this.props as any)['data-is-focusable'] === false || disabled || this._isSplitButton) ? false : true,
+        'data-is-focusable': !((this.props as any)['data-is-focusable'] === false || (disabled && !allowDisabledFocus) || this._isSplitButton) ? false : true,
```

Comment:
I think the new ! at the start of the line is incorrect. Because right now the way this works is if any of those are true, then data-is-focusable should evaluate to false, but with the bang at the front as part of the ternary it reads if any of those conditions are true, then it should be true

Reply:
Yes you are right, thank you for catching this - will correct in a new commit.